app/src/main/java/com/appunite/loudius/di/NetworkModule.kt

```diff
@@ -55,21 +61,21 @@ object NetworkModule {
     @Provides
     @Singleton
     @AuthAPI
-    fun provideAuthRetrofit(
```

Comment:
question: Is this really returning AuthKtor ? Wouldn't it be a better idea to rename it to something like provideHttpClient? It is not something specific to authentication only from what I understand ?

Reply:
Good idea with HttpClient instead of Ktor! I'll change that to `provideAuthHttpClient`. It's specific to authentication, hence the name. We have `BASE_API_URL` and `AUTH_API_URL` and we're using the second one only in authentication. In next PR you will see `provideBaseHttpClient`. :)